benchmarks/nemo/nemolite2d/manual_versions/psykal_cpp/time_step.cpp

```diff
@@ -1,7 +1,4 @@
 #include <iostream>
-#include <chrono>
```

Comment:
These includes no longer needed?

Reply:
No, I realized they are not used and removed them here, even if it has nothing to do with this PR per se